charts/hono/README.md

```diff
@@ -529,3 +529,65 @@ helm install --dependency-update -n hono --set honoImagesType=quarkus-native ecl
 ```
 
 of Quarkus based services images.
+
+
+## Using Kafka based Messaging
+
+The chart can be configured to use Kafka as the messaging network instead of an AMQP 1.0 messaging network.
+The property `messagingNetworkType` is used to select the type of the messaging network.
+
+The following command provides a quickstart for Kafka based messaging:
+
+```bash
+helm install --dependency-update -n hono --set messagingNetworkType=kafka --set kafkaMessagingClusterExample.enabled=true --set amqpMessagingNetworkExample.enabled=false  eclipse-hono eclipse-iot/hono
```

Comment:
How about adding a note here, that `minikube tunnel` needs to be running if using minikube.
I can imagine this being a common mistake when using minikube, with the install command running into a timeout in that case.

Reply:
Done (and rebased).